### PR TITLE
Add rosco package

### PR DESCRIPTION
**Pull request by nikhar-abbas**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by nikhar-abbas** [comment]
@conda-forge-admin, looks like all checks are passed. 
Anything else necessary to get this merged?

File: recipes/rosco/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/nrel/rosco/archive/v{{ version }}.tar.gz
+  sha256: fc4813439be6cbf275cee1fa884e521490559ca87d2f63498ae4fab2dfa9bc48
+
+build:
+  number: 0
+  skip: False  # [win]
```
Comment (marcelotrevisani) [inline recipes/rosco/meta.yaml:14]:
```suggestion
```

Comment (marcelotrevisani) [reply recipes/rosco/meta.yaml:14]: do you want to build it just for ``windows``?

Comment (nikhar-abbas) [reply recipes/rosco/meta.yaml]: removed

Comment (nikhar-abbas) [reply recipes/rosco/meta.yaml]: Would like it to build for linux, osx, and windows

File: recipes/rosco/build.sh [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+mkdir build
+cd build 
+
+cmake \
+    -DCMAKE_Fortran_FLAGS="-ffree-line-length-0" \
+    -DCMAKE_INSTALL_PREFIX=${PREFIX} \
+    ..
+
+make install
```
Comment (wolfv) [inline recipes/rosco/build.sh:11]: can you use `-j${CPU_COUNT}`?

File: recipes/rosco/build.sh [hunk 3]
```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+mkdir build
+cd build 
+
+cmake \
+    -DCMAKE_Fortran_FLAGS="-ffree-line-length-0" \
```
Comment (wolfv) [inline recipes/rosco/build.sh:7]: Maybe add `-DCMAKE_BUILD_TYPE=Release` for Unix as well.

File: recipes/rosco/build.sh (hunk 2 above)
Comment (nikhar-abbas) [reply recipes/rosco/build.sh]: added

File: recipes/rosco/build.sh (hunk 3 above)
Comment (nikhar-abbas) [reply recipes/rosco/build.sh:7]: added

**Pull request merged**